components/filter/test/filter-tags.visual-diff.js

```diff
@@ -64,13 +68,9 @@ describe('d2l-filter-tags', () => {
 	describe('clearable behavior', () => {
 		const selector = '#basic';
 
-		before(async() => {
-			await page.$eval(selector, async(elem) => {
-				elem.parentNode.style.width = '1500px';
-				elem._showHiddenTags = false;
-				await elem.updateComplete;
-			});
-			await page.waitForTimeout(2000);
+		beforeEach(async() => {
+			await page.reload();
```

Comment:
I don't love doing a page reload before each, but this is the simplest way to reset the components back to their initial state so they aren't dependant on each other from one to the next (i.e., currently test 1 deletes last item, test 2 deletes second item, test 2's visual diff appearance relies on result of test 1)

Reply:
Yup that's a good idea